docs/internal.md

```diff
@@ -64,6 +67,46 @@ yarn package
 ```
 
 This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready to be uploaded to stores.
+This version will only connect to _KILT Spiritnet_ blockchain via predefined node endpoints.
+
+### For the internal version
+
+Run the following commands:
+
+```
+yarn install
+yarn build-internal
+yarn package
+```
+
+This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready for developers to use.
+This version will connect to any node endpoints the user inputs and by default to the _KILT Peregrine_ blockchain.
+
+## Uploading to the release's site
+
+We recommend external developers to download the test version of the Sporran extension from [the releases' page on GitHub](https://github.com/BTE-Trusted-Entity/sporran-extension/releases).
```

Comment:
Weird to see recommendations for external developers in the documentation for internal developers

Reply:
This is just to explain why do we need to upload it there.

Reply:
Oh right, that’s fair